frontend/src/pages/Launch/index.tsx

```diff
@@ -63,6 +63,7 @@ export default function LaunchPage() {
 
   return (
     <Row className={styles.wrapper}>
+      <Box as="span" className={`${styles.backgroundContainer} ${styles.background}`} />
```

Comment:
Not sure we want this background in the app too, imo it's better to keep a clean an simple app interface